### ISSUE TITLE
python: command not found --- Conda Environment mandatory?

### ISSUE THREAD

**Issue by Karinon** [FREVA-CLINT/freva-deployment#61]
As a test, I didn't fill out `2. Anaconda installation dir. for core` and also did not enable `3. Install a new Freva anaconda environment`. As it didn't stop me, I thought it was optional.

During the deployment, the job died with:

```
TASK [Installing additional python packages] *************************************************************************************************************
failed: [vaskebjorn2.cloud.dkrz.de] (item=git+https://gitlab.dkrz.de/k204230/install-kernelspec.git) => {"ansible_loop_var": "item", "changed": true, "cmd": "python -m pip install \"git+https://gitlab.dkrz.de/k204230/install-kernelspec.git\"", "delta": "0:00:00.002667", "end": "2024-01-18 09:06:57.430285", "item": "git+https://gitlab.dkrz.de/k204230/install-kernelspec.git", "msg": "non-zero return code", "rc": 127, "start": "2024-01-18 09:06:57.427618", "stderr": "/bin/sh: line 1: python: command not found", "stderr_lines": ["/bin/sh: line 1: python: command not found"], "stdout": "", "stdout_lines": []}
```

`/bin/sh: line 1: python: command not found` is quite a strange error message as I have defined a python-binary in `15. Python path..` and there is definitely a python available on the machine. It looks like this task searches for python somewhere else, I don't know where.

After I have allowed it to install a new Anaconda env and have defined a path under `2.` it worked.

**Comment by antarcticrainforest** [comment]
I see. That's because I assumed that python didn't point anywhere. I'll try to have a look at that.

**Comment by antarcticrainforest** [comment]
The refactor branch uses a micromamba to setup a temp conda. Can you double check if it works for you now?

**Comment by antarcticrainforest** [comment]
Addressed in #105
